graylog2-web-interface/src/views/components/datatable/DataTable.tsx

```diff
@@ -110,11 +137,19 @@ const DataTable = ({
   condensed,
   editing,
 }: Props) => {
-  const formContext = useFormikContext();
+  const formContext = useContext(FormikContext);
   const onRenderComplete = useContext(RenderCompletionCallback);
   const widget = useContext(WidgetContext);
   useEffect(onRenderComplete, [onRenderComplete]);
+  const [rowPivotColumnsWidth, setRowPivotColumnsWidth] = useState<{ [key: string]: number }>({});
```

Comment:
What is the purpose of maintaining the header widths? Actually for me pinning still works without it.

Reply:
Every  pinned column `Cn` should have CSS parameter `left` equal `sum(C1.width...Cn-1.width)`
Which `left` parameter did you put to make it work without width calculation?